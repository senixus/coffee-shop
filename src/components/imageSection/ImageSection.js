import coffeeOne from "../../assets/images/coffee/1.jpg";
import coffeeTwo from "../../assets/images/coffee/2.jpg";
import coffeeThree from "../../assets/images/coffee/3.jpg";
import titleSeperator from "../../assets/images/title-separator.png";
import "./imageSection.scss";

const ImageSection = () => {
  return (
    <section>
      <div className="title-section">
        <h2>The Coffee</h2>
        <img src={titleSeperator} alt="coffee seperator" />
        <p>
          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
          eiusmod tempor incididunt ut labore et dolore
        </p>
      </div>

      <div className="coffee-section container">
        <div className="coffee-section__child">
          <div className="coffee-section__group">
            <img src={coffeeOne} alt="coffee" className="coffee-images" />
            <h3>Coffee</h3>
            <p className="coffee-text">
              Quisque rutrum, aenean imperdiet. Etiam ultricies nisi vel augue.
              Curabitur ullamcorper ultricies nisi, nam eget dui. Etiam rhoncus
              maecenas tempus,
            </p>
          </div>
          <div className="coffee-section__group">
            <img src={coffeeTwo} alt="coffee" className="coffee-images" />
            <h3>Coffee</h3>
            <p className="coffee-text">
              Quisque rutrum, aenean imperdiet. Etiam ultricies nisi vel augue.
              Curabitur ullamcorper ultricies nisi, nam eget dui. Etiam rhoncus
              maecenas tempus,
            </p>
          </div>
          <div className="coffee-section__group">
            <img src={coffeeThree} alt="coffee" className="coffee-images" />
            <h3>Coffee</h3>
            <p className="coffee-text">
              Quisque rutrum, aenean imperdiet. Etiam ultricies nisi vel augue.
              Curabitur ullamcorper ultricies nisi, nam eget dui. Etiam rhoncus
              maecenas tempus,
            </p>
          </div>
        </div>
      </div>
    </section>
  );
};

export default ImageSection;
