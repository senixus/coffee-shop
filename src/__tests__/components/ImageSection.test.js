import { shallow } from "enzyme";
import React from "react";
import ImageSection from "../../components/imageSection/ImageSection";

it("expect to render ImageSection component", () => {
  console.log("Testing");
  const wrapper = shallow(<ImageSection />);
  expect(wrapper.length).toBe(1);
  expect(wrapper).toMatchSnapshot();
});
