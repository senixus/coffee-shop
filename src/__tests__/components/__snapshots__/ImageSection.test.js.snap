// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render ImageSection component 1`] = `
<section>
  <div
    className="title-section"
  >
    <h2>
      The Coffee
    </h2>
    <img
      alt="coffee seperator"
      src="title-separator.png"
    />
    <p>
      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore
    </p>
  </div>
  <div
    className="coffee-section container"
  >
    <div
      className="coffee-section__child"
    >
      <div
        className="coffee-section__group"
      >
        <img
          alt="coffee"
          className="coffee-images"
          src="1.jpg"
        />
        <h3>
          Coffee
        </h3>
        <p
          className="coffee-text"
        >
          Quisque rutrum, aenean imperdiet. Etiam ultricies nisi vel augue. Curabitur ullamcorper ultricies nisi, nam eget dui. Etiam rhoncus maecenas tempus,
        </p>
      </div>
      <div
        className="coffee-section__group"
      >
        <img
          alt="coffee"
          className="coffee-images"
          src="2.jpg"
        />
        <h3>
          Coffee
        </h3>
        <p
          className="coffee-text"
        >
          Quisque rutrum, aenean imperdiet. Etiam ultricies nisi vel augue. Curabitur ullamcorper ultricies nisi, nam eget dui. Etiam rhoncus maecenas tempus,
        </p>
      </div>
      <div
        className="coffee-section__group"
      >
        <img
          alt="coffee"
          className="coffee-images"
          src="3.jpg"
        />
        <h3>
          Coffee
        </h3>
        <p
          className="coffee-text"
        >
          Quisque rutrum, aenean imperdiet. Etiam ultricies nisi vel augue. Curabitur ullamcorper ultricies nisi, nam eget dui. Etiam rhoncus maecenas tempus,
        </p>
      </div>
    </div>
  </div>
</section>
`;
